frontends/web/src/routes/device/deviceswitch.tsx

```diff
@@ -15,20 +15,20 @@
  */
 
 import { Component, h, RenderableProps } from 'preact';
-import { TDevices } from '../../api/devices';
+import { TDevices, TProductName } from '../../api/devices';
 import BitBox01 from './bitbox01/bitbox01';
 import { BitBox02 } from './bitbox02/bitbox02';
 import { BitBox02Bootloader } from '../../components/devices/bitbox02bootloader/bitbox02bootloader';
 import { Waiting } from './waiting';
 
 interface Props {
     devices: TDevices;
-    deviceID: string | null;
+    deviceID?: TProductName;
```

Comment:
deviceID is actually not the product name, it is an arbitrary ID assigned by the backend. please undo this